cmd/zdb/zdb.c

```diff
@@ -747,6 +762,12 @@ usage(void)
 	(void) fprintf(stderr, "    Options to control amount of output:\n");
 	(void) fprintf(stderr, "        -b --block-stats             "
 	    "block statistics\n");
+	(void) fprintf(stderr, "           --bin=(lsize|psize|asize) "
+	    "bin blocks based on this size in all three columns\n");
+	(void) fprintf(stderr,
+	    "           --class=(normal|special|dedup|other)[,...]\n"
+	    "                                     only consider blocks from "
```

Comment:
nit: the indentation here looks wrong.

Reply:
I don't see a problem here, looks and works fine to me. What exactly is wrong?

Reply:
Yup it's good, my mistake.